EFI/OC/config.plist

```diff
@@ -666,25 +666,65 @@
 		<dict>
 			<key>PciRoot(0x0)/Pci(0x1f,0x3)</key>
 			<dict>
+				<key>AAPL,slot-name</key>
+				<string>Internal@0,31,3</string>
+				<key>device-id</key>
+				<data>
+				cKEAAA==
+				</data>
+				<key>device_type</key>
+				<string>Audio device</string>
 				<key>layout-id</key>
 				<data>
 				AwAAAA==
 				</data>
+				<key>model</key>
+				<string>Sunrise Point-LP HD Audio</string>
 			</dict>
 			<key>PciRoot(0x0)/Pci(0x2,0x0)</key>
 			<dict>
 				<key>AAPL,ig-platform-id</key>
 				<data>
 				AADAhw==
 				</data>
+				<key>AAPL,slot-name</key>
+				<string>Internal@0,2,0</string>
 				<key>device-id</key>
 				<data>
-				wIcAAA==
+				FlkAAA==
 				</data>
+				<key>device_type</key>
+				<string>VGA compatible controller</string>
 				<key>enable-hdmi20</key>
 				<data>
 				AQAAAA==
 				</data>
+				<key>framebuffer-con1-enable</key>
+				<data>
+				AQAAAA==
+				</data>
+				<key>framebuffer-con1-type</key>
+				<data>
+				AAgAAA==
+				</data>
+				<key>framebuffer-con2-enable</key>
+				<data>
+				AQAAAA==
```

Comment:
Why adding this section? This is not needed.

Reply:
Sorry late answer. Covid-19 related problems at work. I just pasted hackintosh framebuffer. Don't really know what it does, but allows me to have 2gb dedicated ram to GPU instead of 1.5gb.

Reply:
Where do you need 2GB for GPU? And from WEG manual - https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md framebuffer-unifiedmem (VRAM, it is not recommended to use this patch).